src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4900,9 +4936,9 @@ private bool IsConstructorAccessible(MethodSymbol constructor, ref HashSet<Diagn
         }
 
         protected BoundExpression BindClassCreationExpression(
-            CSharpSyntaxNode node,
+            SyntaxNode node,
```

Comment:
>SyntaxNode [](start = 12, length = 10)

why change?

Reply:
we're passing `Syntax` from a bound node.